### PR TITLE
[FE-65] Hall of fame imporvement

### PR DESCRIPTION
**Pull request by AnCichocka**
student view:
image

teacher view:
image

### CODE REVIEW

File: frontend/src/router/routes.tsx
Comment (AnCichocka) [inline frontend/src/router/routes.tsx:1]: I cannot make it start with big letter anymore xD

File: frontend/src/components/hallOfFame/StatisticsBox.tsx
Comment (AnCichocka) [inline frontend/src/components/hallOfFame/StatisticsBox.tsx:1]: placeholder for statistics

**Comment by AnCichocka** [comment]
@Szyntos please, checkout on this branch and test

**Review by Szyntos — APPROVED** [review]
Looks good, ngl very nice work there

Tested manually, i haven't found any bugs, soooo

**Pull request merged**